### PR TITLE
Enabled solr in eZ Commerce on Platform.sh

### PR DESCRIPTION
**Pull request by vidarl**
This PR enables solr by default as it is required by eZ Commerce

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -119,6 +119,14 @@ crons:
...
         spec: "* * * * *"
         cmd: "php bin/console ezplatform:cron:run"
+    refresh:
+        # Checks for changes and refresh cache
+        spec: "*/5 * * * *"
+        cmd: "php bin/console silversolutions:cache:refresh"
+    writestat:
+        # Calculate statistical data for active sessions
+        spec: "*/5 * * * *"
+        cmd: "php bin/console silversolutions:sessions --env=prod write_stat"
```
Comment (vidarl) [inline .platform.app.yaml]: The doc must be wrong here. It claims

    */5 * * * * cd /var/www/my_project &&  /usr/bin/php bin/console silversolutions:sessions --env=dev -session_command write_stat

No reason why it should be run with `--env=dev`, is there?
And `-session_command ` should be removed from doc I think

**Comment by vidarl** [comment]
FYI : Havn't been able to test this branch as I get 

```
File error: RecursiveDirectoryIterator::__construct(): Unable to find the wrapper "phar" - did you forget to enable it when you configured PHP?
Script @php bin/console ezplatform:install ezcommerce_installer handling the ezplatform-install event returned with error code 1
Script @ezplatform-install was called via ezcommerce-install```

That problem has been fixed but I guess it is not available in a version required by 2.3 branch of eZ Commerce

**Comment by andrerom** [comment]
please move all PR's to advance repo, we plan to delete this one.

**Comment by vidarl** [comment]
> please move all PR's to advance repo, we plan to delete this one.

Ok, I was not aware that a decision was made. Good that it has been taken.

**Comment by andrerom** [comment]
@vidarl: Also you don’t need to bother with 2.3, it’s 2.4 + master we aim for.

**Comment by vidarl** [comment]
Yeah, but there is no 2.4 branch here. That's why it was against 2.3....

**Pull request closed without merging**